assets/src/edit-story/components/panels/sizePosition/panel.js

```diff
@@ -165,7 +165,7 @@ function SizePositionPanel({
     <SimplePanel name="size" title={__('Size & position', 'web-stories')}>
       {isMedia && isSingleElement && (
         <Row expand>
-          <Button onClick={handleSetBackground} fullWidth>
+          <Button onClick={handleSetBackground} fullWidth tabIndex="-1">
```

Comment:
This is most definitely not the purpose of the underlying ticket - we can't just disable all a11y in this very essential panel! The reason for this is to help non-mouse users to be able to manipulate the elements when they can't use a mouse. Now they can't manipulate elements at all!

Reply:
Thanks for pointing it out. That's right, rolled back and redo the implementation.